railties/test/code_statistics_test.rb

```diff
@@ -17,4 +17,18 @@ def teardown
       @code_statistics = CodeStatistics.new(['tmp dir', @tmp_path])
     end
   end
+
+  test 'ignores hidden files' do
+    content = <<-'CODE'
+      def foo
+        puts 'foo'
+      end
+    CODE
+
+    path = File.join(@tmp_path, '.example.rb')
+    File.write path, content
+    stats = CodeStatistics.new(['hidden file', @tmp_path])
+
+    assert_equal 0, stats.instance_variable_get("@statistics")['hidden file'].lines
```

Comment:
Nice, much cleaner!
Sorry for my ignorance but I'm curious to understand why an `assert_nothing_raised` should be tested here? Should the test focus on checking if the file was processed instead?